std/conv.d

```diff
@@ -2862,7 +2862,17 @@ unittest
 
     // min and max
     real r = to!real(to!string(real.min_normal));
-    assert(to!string(r) == to!string(real.min_normal));
+    version(NetBSD)
+    {
+        // NetBSD notice
+        // to!string returns 3.3621e-4932L. It is less than real.min_normal and it is subnormal value
+        // Simple C code 
+        //     long double rd = 3.3621e-4932L;
+        //     printf("%Le\n", rd);  
+        // has unexpected result: 1.681050e-4932 
+    } else {
+        assert(to!string(r) == to!string(real.min_normal));
+    }
```

Comment:
Per the [style guide](http://dlang.org/dstyle.html), braces need to go on their own line. So, assuming that this code gets merged with the assertion versioned out, then the `else` needs to be put on its own line, and you can just get rid of the extra braces.